crates/core/src/db/db_metrics/mod.rs

```diff
@@ -129,6 +129,11 @@ metrics_group!(
         #[help = "For a given module, the size of its log file (in bytes)"]
         #[labels(db: Address)]
         pub module_log_file_size: IntGaugeVec,
+
+        #[name = spacetime_request_round_time]
+        #[help = "The total time it takes for request to complete"]
+        #[labels(txn_type: WorkloadType, database_address: Address, reducer_symbol: str)]
```

Comment:
Is it important to be able to distinguish this metric by reducer? Or do we just care about the aggregate?

Reply:
I think if we have other metrics to know time spent on indivisual reducers then it can be avoided here.
@cloutiertyler, what's your thoughts on this?

Reply:
I think we care about individual ones because some reducers will have much higher latency than others.